### ISSUE TITLE
[Bug] LSPosed not opening

### ISSUE THREAD

**Issue by drakulaboy** [LSPosed/LSPosed#1964]
### Steps to reproduce/复现步骤

after opening one or a few times, or force kill app, it doesn't open, only after reboot of smartphone and again if i open it a few times then it won't open

### Expected behaviour/预期行为

open LSPosed

### Actual behaviour/实际行为

it says "Shell keeps stopping"

### Xposed Module List/Xposed 模块列表

```shell
none
```

### Magisk Module List/Magisk 模块列表

```shell
LSposed Zygisk
```

### LSPosed version/LSPosed 版本

6562

### Android version/Android 版本

12

### Magisk version/Magisk 版本

24

### Riru version/Riru 版本

N/A

### Version requirement/版本要求

- [X] I am using latest debug CI version of LSPosed and enable verbose log/我正在使用最新 CI 调试版本且启用详细日志

### Logs/日志

[bugreport-spark_vince-SP2A.220505.002-2022-05-27-14-52-06.zip](https://github.com/LSPosed/LSPosed/files/8786450/bugreport-spark_vince-SP2A.220505.002-2022-05-27-14-52-06.zip)

**Comment by yujincheng08** [comment]
Duplicate of #1919

**Comment by drakulaboy** [comment]
hmm, i found out that only on android 12L is this issue, just tried latest zygisk version on 12 only and it's not crashing
